### PR TITLE
Issue# 1215 Correct discription for Swift for configuration doc

### PR DESCRIPTION
**Pull request by xinxiaogang**
Swift auth failed if no tenant information provided. So change tenant
field required as 'yes'.

https://github.com/docker/distribution/issues/1215

### CODE REVIEW

**Comment by xinxiaogang** [comment]
@GordonTheTurtle What's the next step I need do?

File: docs/configuration.md [hunk 1]
```diff
@@ -801,10 +801,10 @@ This storage backend uses Openstack Swift object storage.
       <code>tenant</code>
     </td>
     <td>
-      no
+      --
     </td>
     <td>
-      Your Openstack tenant name.
+      Your Openstack tenant name. Whether required depends on the authentication version you use. More information, please refer to [OpenStack Identity Service](http://docs.openstack.org/developer/keystone/)
```
Comment (stevvooe) [inline docs/configuration.md]: `Required depending the Swift authentication version in use. Please refer to [OpenStack Identity Service](http://docs.openstack.org/developer/keystone/) for more information.`

**Comment by stevvooe** [comment]
@xinxiaogang Make sure to sign your commits. Directions are above.

**Comment by dmp42** [comment]
ping @xinxiaogang  :)

**Pull request closed without merging**